src/pretix/plugins/sendmail/views.py

```diff
@@ -29,6 +29,12 @@ class SenderView(EventPermissionRequiredMixin, FormView):
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['event'] = self.request.event
+        from_log_id = self.request.GET.get('from_log')
+        message = LogEntry.objects.get(id=from_log_id).display()
```

Comment:
Everything works perfectly except the query it return a invalid id
These are the values:
from_log_id = 1
self.request.event = 2018
action_type = 'pretix.plugins.sendmail.sent'

the from_log_id is got from the history template as the URL query param. I can see the sql from the DoDT tab but couldn't look for the query for the from_log_id. Can you show me how can I check if the record is in the DB?